src/tests/owners.rs

```diff
@@ -93,12 +93,8 @@ fn new_crate_owner() {
     token.enqueue_publish(crate_to_publish).good();
 
     // Add the second user as an owner (with a different case to make sure that works)
-    let user2 = app.db_new_user("Bar");
-    token.add_user_owner("foo_owner", "BAR");
```

Comment:
this appears to be testing the case where the invitation is using uppercase while the username is using a different style. if we went with this refactoring then I think we would lose that check, no?

Reply:
agreed... is it fine if I uppercase the user on the helper (and add the case comment)